src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,106 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
 import './FindMovie.scss';
 
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie = ({ addMovie }) => {
+  const [value, setSearchTerm] = useState('');
+  const [results, setSearchResults] = useState({
+    title: '',
+    description: '',
+    imgUrl: '',
+    imdbUrl: '',
+    imdbId: '',
+  });
+  const [filmNotFound, setNotFound] = useState(false);
+
+  const handleChange = (event) => {
+    event.preventDefault();
+    setSearchTerm(event.target.value);
+    setNotFound(true);
+  };
+
+  const findMovie = () => {
+    const API_URL = `https://www.omdbapi.com/?apikey=1e23943c&t=`;
+
+    const result = () => fetch(`${API_URL}${value}`)
+      .then(response => response.json());
+
+    result().then(movies => setSearchResults({
+      title: movies.Title,
+      description: movies.Plot,
+      imgUrl: movies.Poster,
+      imdbUrl: movies.ImdbUrl,
+      imdbId: movies.imdbID,
+    }));
 
-      <div className="field is-grouped">
-        <div className="control">
-          <button
-            type="button"
-            className="button is-light"
-          >
-            Find a movie
-          </button>
+    if (value === '') {
+      setNotFound(false);
+    }
+
+    setSearchTerm('');
+  };
+
+  return (
+    <>
+      <form className="find-movie">
+        <div className="field">
+          <label className="label" htmlFor="movie-title">
+            Movie title
+          </label>
+
+          <div className="control">
+            <input
+              type="text"
+              id="movie-title"
+              placeholder="Enter a title to search"
+              className={classNames(`input`, { 'is-danger': !filmNotFound })}
```

Comment:
```suggestion
              className={classNames(`input`, { 'is-danger': filmNotFound })}
```
<img width="415" alt="Снимок экрана 2021-06-08 в 10 15 01" src="https://user-images.githubusercontent.com/66570142/121141305-7c609300-c843-11eb-945f-eb2b826ee742.png">

Now you have an error message by default even on the app start and it does not work as suppose. Let's fix this